sdfformat on Ubuntu

I can't get sdformat to install on Linux with instructions on website.  I did:

```bash
apt-get -y install libtinyxml2-dev libxml2-utils pkg-config ruby-dev ruby wget
apt-get -y install libignition-math4-dev
cd /usr/src
wget http://osrf-distributions.s3.amazonaws.com/sdformat/releases/sdformat-10.5.0.tar.bz2
tar -xvjf sdformat-10.5.0.tar.bz2
rm -f sdformat-10.5.0.tar.bz2
mkdir build
cd build
cmake ..
```

But get


```
====== Finding 3rd Party Packages ======
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Building for arch: x86_64
-- Found TinyXML2: /usr/lib/x86_64-linux-gnu/libtinyxml2.so  
-- Checking for module 'urdfdom>=1.0'
--   No package 'urdfdom' found
CMake Warning at cmake/SearchForStuff.cmake:34 (find_package):
  By not providing "Findurdfdom.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "urdfdom", but
  CMake did not find one.

  Could not find a package configuration file provided by "urdfdom" with any
  of the following names:

    urdfdomConfig.cmake
    urdfdom-config.cmake

  Add the installation prefix of "urdfdom" to CMAKE_PREFIX_PATH or set
  "urdfdom_DIR" to a directory containing one of the above files.  If
  "urdfdom" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:116 (include)


-- Couldn't find urdfdom >= 1.0, using internal copy
CMake Warning at cmake/SearchForStuff.cmake:55 (find_package):
  By not providing "Findignition-tools.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "ignition-tools", but CMake did not find one.

  Could not find a package configuration file provided by "ignition-tools"
  with any of the following names:

    ignition-toolsConfig.cmake
    ignition-tools-config.cmake

  Add the installation prefix of "ignition-tools" to CMAKE_PREFIX_PATH or set
  "ignition-tools_DIR" to a directory containing one of the above files.  If
  "ignition-tools" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:116 (include)


-- 	ignition-tools not found, for command line utilities and for generating aggregated SDFormat descriptions for sdformat.org, please install ignition-tools.
-- Could NOT find PY_psutil (missing: PY_PSUTIL) 
-- 	Python psutil package not found. Memory leak tests will be skipped
-- Found ruby executable: /usr/bin/ruby
CMake Error at cmake/SearchForStuff.cmake:107 (find_package):
  By not providing "Findignition-cmake2.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "ignition-cmake2", but CMake did not find one.

  Could not find a package configuration file provided by "ignition-cmake2"
  (requested version 2.3) with any of the following names:

    ignition-cmake2Config.cmake
    ignition-cmake2-config.cmake

  Add the installation prefix of "ignition-cmake2" to CMAKE_PREFIX_PATH or
  set "ignition-cmake2_DIR" to a directory containing one of the above files.
  If "ignition-cmake2" provides a separate development package or SDK, be
  sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:116 (include)


-- Configuring incomplete, errors occurred!
```

I can update the README, but can someone comnment on how they got past this? I hope it does not involve also installing urfdom from source etc...

--- a/README.md
+++ b/README.md
@@ -50,9 +50,6 @@ sudo apt-get update
 # Install SDFormat dependencies
 sudo apt-get install -y libtinyxml2-dev liburdfdom-dev libignition-cmake2-dev libignition-tools-dev libignition-math6-dev
 
-# sdformat requires libignition-math
-sudo apt-get install libignition-math4-dev
-
 # Set the version to install
 export GTD_SDFormat_VERSION="10.5.0"
 

--- a/README.md
+++ b/README.md
@@ -21,12 +21,12 @@ GTDynamics uses the SDFormat parser to parse SDF/URDF files containing robot des
 
 ### Homebrew
 
-If you are on a Mac, using Homebrew is the easiest way to get SDFormat installed and it also makes switching versions straightforward.
+Using Homebrew is the easiest way to get SDFormat installed and it also makes switching versions straightforward.
 
 ```sh
 $ # Install homebrew.
 $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-$ # Download sdformat to your preferred location.
+$ # Set up the tap ind install sdformat10
 $ brew tap osrf/simulation
 $ brew install sdformat10
 ```
@@ -40,7 +40,15 @@ We provide an Ubuntu-based process below. Please reference [this tutorial](http:
 
 ```sh
 # Install basic dependencies
-sudo apt-get install ruby-dev build-essential libtinyxml-dev libboost-all-dev cmake pkg-config
+sudo apt-get install -y ruby-dev build-essential libboost-all-dev cmake pkg-config wget lsb-release
+
+# Setup the repo
+sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
+sudo apt-get update
+
+# Install SDFormat dependencies
+sudo apt-get install -y libtinyxml2-dev liburdfdom-dev libignition-cmake2-dev libignition-tools-dev libignition-math6-dev
 
 # sdformat requires libignition-math
 sudo apt-get install libignition-math4-dev
@@ -56,7 +64,7 @@ tar -xvjf sdformat-${GTD_SDFormat_VERSION}.tar.bz2
 cd sdformat-${GTD_SDFormat_VERSION}
 mkdir build && cd build
 
-cmake -DCMAKE_INSTALL_PREFIX ../install ..
+cmake -DCMAKE_INSTALL_PREFIX=../install ..
 make -j4
 sudo make install
 ```

